whistl/combine_metadata.py

```diff
@@ -0,0 +1,38 @@
+'''This script combines metadata files associated with refine.bio datasets. This allows us to
+label additional datasets while keeping all the data in a single version controlled file.
+'''
+import argparse
+import json
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='This script combines metadata files associated '
+                                     'with refine.bio datasets. This allows us to '
+                                     'label additional datasets while keeping all the data in a '
+                                     'single version controlled file.')
+
+    parser.add_argument('file1', help='The path to the first file to combine')
+    parser.add_argument('file2', help='The path to the second file to combine')
```

Comment:
What are the two files that you're expecting to combine here?

Reply:
That's a good point, I don't think I explain which file I'm specifically talking about anywhere in this script. I'm combining instances of the `aggregated_metadata.json` file that is present in each refine.bio dataset download.

I'll update the code to be more descriptive.